### PR TITLE
Stay logged in

### PR DESCRIPTION
**Pull request by stefanalex99**

### CODE REVIEW

File: interface/views.py [hunk 1]
```diff
@@ -27,6 +27,9 @@
 
 
 def login_view(request):
+    if (request.user.is_authenticated):
```
Comment (Jokeswar) [inline interface/views.py]: No parenthesis please.

File: interface/admin.py [hunk 2]
```diff
@@ -13,16 +13,21 @@
 
 @admin.register(Assignment)
 class AssignmentAdmin(admin.ModelAdmin):
-    actions = ['download_submissions']
+    actions = ['download_submissions', 'download_all_submissions']
```
Comment (Jokeswar) [inline interface/admin.py]: This part is the same as #46. I suggest using `git rebase --onto master origin/downloadAllSubmissions  StayLoggedIn` to keep only the logged in part. You will get conflicts when we will try to merge this branch after #46, or the other way around.

Comment (stefanalex99) [reply interface/admin.py]: I am not sure if I solved the problem, can you check?

Comment (Jokeswar) [reply interface/admin.py]: It didn't solve it. You can check here to see the differences: https://github.com/vmck/acs-interface/pull/47/files

**Pull request merged**